frontend/src/app/data-entry/source/source.component.ts

```diff
@@ -35,7 +35,7 @@ export class SourceComponent {
 
     public sourceTitle = toSignal(
         this.source$.pipe(
-            map((source) => `${source.medievalAuthor}, ${source.medievalTitle}`)
+            map((source) => source.name)
```

Comment:
In that case, we don't need to query `medievalAuthor` and `medievalTitle` anymore.